sdk/python/kfp/_client.py

```diff
@@ -52,21 +55,23 @@ def __init__(self, host=None, client_id=None, namespace='kubeflow'):
     """
 
     self._host = host
```

Comment:
host or os.environ.get(KF_PIPELINES_ENDPOINT_ENV)?

Reply:
its a simple/standard way to say `host if host else os.env...`

Reply:
I mean if we should set _host to be the value from env var if it's missing? Either it's from parameter or env var, it should be the same thing.

Reply:
@hongye-sun i consolidated `self._host` and `self._uihost` since `_host` wasnt used, now if no env var and `host` is provided it will use `host` (still not clear, how `host` will open up the browser on the UI if its the remote API endpoint)